src/index.ts

```diff
@@ -983,6 +983,7 @@ export type RetryParametersObj = {
 		attempt: number,
 		maxAttempts: number,
 	) => void;
+	getRetryAfterHeader?: (err: unknown) => string | undefined;
```

Comment:
An alternative could have been to change `onRetry` to be returning the retry after header.